ui/app/clinical/displaycontrols/patientContext/directives/patientContext.js

```diff
@@ -44,6 +44,18 @@ angular.module('bahmni.clinical')
             });
         };
 
+        var translateAttributes = function (attributes) {
+            _.forEach(attributes, function (attribute, key) {
+                var keyPrefix = "";
+                var keyName = key.toUpperCase().replace(/\s\s+/g, ' ').replace(/[^a-zA-Z0-9 _]/g, "").trim().replace(/ /g, "_");
```

Comment:
@buvaneswari-arun Can we move keyName regression to a util as its being used in multiple places